### ISSUE TITLE
FF: code.org/hourofcode/overview text overlaps

### ISSUE THREAD

**Issue by tanyaparker** [code-dot-org/dance-party#371]
![image](https://user-images.githubusercontent.com/1920530/48453402-10d7e700-e768-11e8-8d24-d29de05948ed.png)

**Comment by simplycloud** [comment]
@madelynkasula @boatmarina do we want this into our final CSED release?

**Comment by madelynkasula** [comment]
@simplycloud no, i don't think this is urgent enough. this is in progress now, but likely won't get in for today's deploy

**Comment by boatmarina** [comment]
yeah if this isn't ready today, i think we can hold off until post-HOC to
fix.
